ui/js/controllers/perf/graphs.js

```diff
@@ -43,20 +43,11 @@ perf.controller('GraphsCtrl', [
 
         $scope.createAlert = function (dataPoint) {
             $scope.creatingAlert = true;
-            PhAlerts.createAlert(dataPoint).then(function (alertSummaryId) {
-                PhAlerts.getAlertSummaries({
-                    seriesSignature: dataPoint.series.seriesSignature,
-                    repository: dataPoint.project.id,
-                }).then(function (alertSummaryData) {
-                    $scope.creatingAlert = false;
-
-                    var alertSummary = alertSummaryData.results.find(result =>
-                        result.id === alertSummaryId);
-                    $scope.tooltipContent.alertSummary = alertSummary;
-
-                    dataPoint.series.relatedAlertSummaries = alertSummaryData.results;
-                    plotGraph();
-                });
+            PhAlerts.createAlert(dataPoint)
+            .then(alertSummaryId => refetchGraphData(alertSummaryId, dataPoint))
```

Comment:
Can you rename `refetchGraphData` to `refreshGraphData` and put the call to `plotGraph` inside it? lgtm aside from that.

Reply:
@wlach I updated.